scripts/zones/Phomiuna_Aqueducts/Zone.lua

```diff
@@ -8,16 +8,16 @@ require('scripts/globals/status')
 local zoneObject = {}
 
 zoneObject.onInitialize = function(zone)
+    -- NM Persistence
+    GetMobByID(ID.mob.TRES_DUENDES):setLocalVar("cooldown", os.time() + math.random(900, 1800))
+
```

Comment:
I dont think you need this anymore since you added the persist on the bottom which includes the respawn time if its alive and server crashes.